projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -381,6 +381,93 @@ public function get_item_schema() {
 		return $this->add_additional_fields_schema( $this->schema );
 	}
 
+	/**
+	 * Updates the item.
+	 * Overrides the parent method to resend the email when the item is updated from spam to publish.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function update_item( $request ) {
```

Comment:
I didn't look if we check earlier things like `id` isn't empty/invalid, post with that ID exists etc but feels like something like that is missing here?

Reply:
You're right on this one. I'm mostly relying on the parent's process to validate, but I'm also taking the `id` first.

Ref: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qvapyhqrf%2Serfg%2Qncv%2Sraqcbvagf%2Spynff%2Qjc%2Qerfg%2Qcbfgf%2Qpbagebyyre.cuc%3Se%3Qq13qsq59%26zb%3Q235%26sv%3Q17%23932-og

I think I'll invert those, so the parent process continues to handle validity. Something like:
```php
$updated_item = parent::update_item( $request );

if ( ! is_wp_error( $updated_item ) && ! empty( $updated_item->data && ! empty( $updated_item->data['status'] ) ) ) {
	$post_id         = $request['id'];
	$previous_status = get_post_status( $post_id );
...
```